packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -0,0 +1,142 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import {Input} from 'react-vapor';
+
+export interface ILimitOwnProps {
+    title: string;
+    limitLabel?: string;
+    usageNumber?: number;
+    limitNumber?: number;
+    isLimitModifiable?: boolean;
+    isLimitTheGoalToReach?: boolean;
+    isHistoryIncluded?: boolean;
+}
+
+export interface ILimitDispatchProps {
+    onRender?: () => void;
+    onDestroy?: () => void;
+}
+
+export interface ILimitProps extends ILimitOwnProps, ILimitDispatchProps {}
+
+export class Limit extends React.Component<ILimitProps, {}> {
+    static defaultProps: Partial<ILimitProps> = {
+        limitLabel: 'Limit',
+        limitNumber: 100,
+        isLimitModifiable: false,
+        isLimitTheGoalToReach: false,
+        isHistoryIncluded: false,
+    };
+
+    componentDidMount() {
+        this.props.onRender?.();
+    }
+
+    componentWillUnmount() {
+        this.props.onDestroy?.();
+    }
+
+    render() {
+        const mainLimitBoxClasses = classNames('limit-box-main', 'p2', 'pb1');
+
+        return (
+            <div className="limit-box mb2">
+                <div className={mainLimitBoxClasses}>
+                    {this.getHeaderDivision()}
+                    {this.getContentDivision()}
+                </div>
+                {this.getProgressBar()}
+            </div>
+        );
+    }
+
+    private getHeaderDivision() {
+        const titleClasses = classNames('flex', 'space-between');
+        return (
+            <div className={titleClasses}>
+                <label className="form-control-label"> {this.props.title}</label>
+                {this.getHistoryIcon()}
+            </div>
+        );
+    }
+
+    private getHistoryIcon() {
+        const iconClasses = classNames('icon', 'mod-lg', 'limit-history-button');
+        return (
+            this.props.isHistoryIncluded && (
+                <span className={iconClasses}>
+                    <span className="fill-medium-blue">
+                        <svg className="" viewBox="0 0 16 16">
```

Comment:
There an SVG component [here](https://github.com/coveo/react-vapor/tree/master/packages/react-vapor/src/components/svg)

Reply:
Use the `<Svg />` component